### PR TITLE
docs: Added rejected takeoff page

### PR DESCRIPTION
**Pull request by SweatyCam**
## Summary
Rejected takeoff page for the advanced guides

### Location
https://docs-git-fork-sweatycam-rejected-take-off-flybywire.vercel.app/pilots-corner/advanced-guides/rejected-take-off/

Discord username (if different from GitHub): Sweaty Cam#3177

### CODE REVIEW

**Comment by Valastiri** [comment]
Any update on this PR?

**Comment by SweatyCam** [comment]
> Any update on this PR?

yes sorry, been way busy with work/uni, barely have time to fly. will update this soon

File: docs/pilots-corner/advanced-guides/rejected-take-off.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Rejected Takeoff in the A320
+
+The term rejected takeoff (also known as RTO) refers to incidents in which an aircraft's pilots elect to abort their departure before reaching a critical speed. This critical speed is known as 'V1', and this refers to the maximum speed an aircraft can reach on its takeoff roll and then be brought to a stop whilst remaining within the ASDA (Accelerate-Stop Distance Available). If the aircraft reaches this speed and the Pilot Flying has not yet taken action to stop the plane, the takeoff must be continued unless the airplane is unsafe to fly.  
+
+When the aircraft speed is at or above 100 kt, it may become hazardous to reject a takeoff, therefore a takeoff is split into 3 phases:  
+
+- Below 100 kt  
+- Above 100 kt and below V1
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/rejected-take-off.md]: I believe abbreviation in relation "knots" is `kts` as we've used in various other portions of our documentation. Double check the page and reference accordingly.

File: docs/pilots-corner/advanced-guides/rejected-take-off.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        - ENG REVERSE UNLOCKED
+        - ENG 1(2) THR LEVER FAULT
+- Above V1:  
+    - Malfunctions or conditions that give unambiguous indications that the aircraft will not fly safely
+
+## Rejected Takeoff Procedure  
+
+!!! note
+
+      Because in the simulator most of the time you are on your own, the procedure has been condensed for a single pilot to perform. In real life operations, the procedures are split between the Captain and FO.
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/rejected-take-off.md]:
```suggestion
!!! note

      We have modified and condensed the rejected takeoff procedure to assist a single pilot to perform for simulation purposes. In real life operations, the procedures are split between the Captain and FO. 
```

**Review by Valastiri — CHANGES_REQUESTED** [review]
some minor changes. Will verify use of `kt` vs `kts`.

File: docs/pilots-corner/advanced-guides/rejected-take-off.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## What Happens During a Rejected Takeoff in Real Life?
+
+The decision to reject the takeoff and the stop action is made by the Captain, it is therefore recommended that the Captain keeps his hand on the thrust levers until the aircraft reaches V1, whether he is Pilot Flying (PF) or Pilot Not Flying (PNF).
+
+As soon as the Captain decides to abort, he calls “STOP”, takes over control of the aircraft, and performs the stop actions.
+
+## Reasons for a Rejected Takeoff  
+
+It is not possible to list all the factors that could lead to the decision to reject the takeoff. However, in order to help the Captain make a decision, the ECAM inhibits warnings that are not essential from 80 kt to 1500 ft (or 2 min after lift-off, whichever occurs first). When this occurs, a magenta T.O INHIBIT memo is displayed in the Upper ECAM (E/WD).
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/rejected-take-off.md]:
```suggestion
It is not possible to list all the factors that could lead to the decision to reject the takeoff. However, in order to help the Captain make a decision, the ECAM inhibits warnings that are not essential passing 80 kts or 1500 ft (or 2 min after lift-off, whichever occurs first). When this occurs, a magenta T.O INHIBIT memo is displayed in the Upper ECAM (E/WD).
```

Please verify this as the statement is confusing in the original text. "not essential from 80 kt to 1500 ft" are two different units so using "to" is where the confusion happens. 

If the suggestion reads better to you just commit via github.

File: docs/pilots-corner/advanced-guides/rejected-take-off.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+[//]: # (CABIN CREW.....................................................ALERT  )
+
+[//]: # (ECAM ACTIONS..........................................PERFORM  )
+
+---
+
+!!! note 
+
+    - Full reverse thrust may be used until coming to a complete stop. But, if there is enough runway available at the end of the deceleration, it is preferable to reduce reverse thrust when passing 70 kts.
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/rejected-take-off.md]: Is there a thrust setting that is preferrable? i.e. reduce reverse thrust to idle -- or something else?

**Review by Valastiri — CHANGES_REQUESTED** [review]
Also just to confirm images are all optimized yes?

**Review by Valastiri — APPROVED** [review]

**Comment by Valastiri** [comment]
check build

File: docs/pilots-corner/advanced-guides/rejected-take-off.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+# Rejected Takeoff in the A320
+
+The term rejected takeoff (also known as RTO) refers to incidents in which an aircraft's pilots elect to abort their departure before reaching a critical speed. This critical speed is known as 'V1', and this refers to the maximum speed an aircraft can reach on its takeoff roll and then be brought to a stop whilst remaining within the ASDA (Accelerate-Stop Distance Available). If the aircraft reaches this speed and the Pilot Flying has not yet taken action to stop the plane, the takeoff must be continued unless the airplane is unsafe to fly.
```
Comment (donstim) [inline docs/pilots-corner/advanced-guides/rejected-take-off.md:3]: Sorry, just finally looked at this, but the relevant distance is the accelerate-stop distance (the distance the airplane actually needs to accelerate and stop in accordance with the regulations). The accelerate-stop distance available (ASDA) is the length of runway and stopway that is available for an accelerate-stop and can be longer than the accelerate-stop distance.

Comment (SweatyCam) [reply docs/pilots-corner/advanced-guides/rejected-take-off.md:3]: > Sorry, just finally looked at this, but the relevant distance is the accelerate-stop distance (the distance the airplane actually needs to accelerate and stop in accordance with the regulations). The accelerate-stop distance available (ASDA) is the length of runway and stopway that is available for an accelerate-stop and can be longer than the accelerate-stop distance.

Right on, I’ll get that changed tonight. Unless someone is able to change it and merge it themselves?

Comment (Valastiri) [reply docs/pilots-corner/advanced-guides/rejected-take-off.md:3]: I can probably correct this quickly. In an hour.

Comment (Valastiri) [reply docs/pilots-corner/advanced-guides/rejected-take-off.md:3]: Had a small snafu in my dev environment :/ corrective PR made already.

**Pull request merged**